yt/yt/ytlib/api/native/client_authentication_impl.cpp

```diff
@@ -117,16 +127,22 @@ TIssueTokenResult TClient::DoIssueTokenImpl(
     const TIssueTokenOptions& options)
 {
     auto tokenHash = GetSha256HexDigestLowerCase(token);
+    auto tokenPrefix = token.substr(0, TokenPrefixLength);
+    if (!tokenPrefix.StartsWith("ytct-")) {
```

Comment:
Maybe add `constexpr TStringBuf CypressTokenPrefix = "ytct-"` somewhere?

Reply:
Added CypressTokenRegex to better handle specific tokens